### DISCUSSION TITLE
Squiggle errors only

### DISCUSSION THREAD

**Discussion by edgardozoppi** [James-Yu/LaTeX-Workshop#3369] [Q&A]
Hi, I would like to know if there is a way to configure latex-workshop to only squiggle errors and ignore warnings and messages.

Thanks!

**Comment by James-Yu** [comment]
Yes.

```
        "latex-workshop.message.information.show": {
          "type": "boolean",
          "default": false,
          "markdownDescription": "Display information messages in popup notifications."
        },
        "latex-workshop.message.warning.show": {
          "type": "boolean",
          "default": true,
          "markdownDescription": "Display warning messages in popup notifications."
        },
```

**Comment by edgardozoppi** [comment]
@James-Yu , I tried both settings but I think they are not what I am looking for. I think they are related with showing popup notifications but I asked about the squiggles in the .tex files.

I would like to see all the errors, warnings and messages in the problems panel but only highlight errors (underlining with squiggles) in the .tex files. Right now the squiggles appear also for warnings and messages and there is no way I am aware of to prevent this behavior.

Thanks

**Comment by James-Yu** [comment]
Unfortunately, this (show underline or not) cannot be controlled by the extension. You may refer to vscode configs for details, yet I suspect that the editor does not allow you to achieve what you would like to.

**Comment by Savvylion** [comment]
It would be very nice to have this feature, especially if you are editing dependent tex files for content and know the warnings will disappear when the full document is compiled.
